src/videopython/understanding/transcription.py

```diff
@@ -0,0 +1,35 @@
+from typing import Literal
+
+import whisper
+
+from videopython.base.video import Video
+
+
+class Transcription:
```

Comment:
I'm not a big fan of naming this class `Transcription` as it sounds very passive whereas it's more of a class, which transcribes the video. What I would love is to have a `Transcription` dataclass, which would contain all the `Segment` classes per video. Something like this:
```
@dataclass
class Transcription:
    segments: list[TranscriptionSegment]

@dataclass
class TranscriptionSegment:
    start: float
    end: float
    text: str
```

What about naming this something like `VideoToTranscription`? In a similar way that we have e.g. `TextToVideo` in the generation module.

Reply:
Do you think that creating two data classes is useful here instead of quite straightforward structure with simple dictionary?

Reply:
Hmm I would even go a stop further tbh - I would define the `Transcription` datalcass somewhere `videopython.base`. It'll be benefitial to have a proper structure for it if we'll use it in multiple places in the codebase instead of just using a dict imo.

Reply:
Added but I don't like `VideoToTranscription` name here. I think `VideoTranscription` fits better.